Quran/SettingsCreators.swift

```diff
@@ -58,6 +62,16 @@ class NavigationSettingsCreators: SettingsCreators {
             }
             Analytics.shared.review(automatic: false)
         }
-        return [translation, audio, review]
+        let email = Setting(name: l("email_us"), image: #imageLiteral(resourceName: "email-outline")) { vc in
+            if MFMailComposeViewController.canSendMail() {
+                let mail = MFMailComposeViewController()
+                mail.navigationBar.tintColor = .white
+                mail.mailComposeDelegate = self.parentController
```

Comment:
This Delegate is implemented in `SettingsViewController` to dismiss.

Reply:
👍